bindings/rust/src/bindings/mod.rs

```diff
@@ -587,16 +584,22 @@ mod tests {
 
         blobs.pop();
 
-        let error =
-            KZGProof::verify_blob_kzg_proof_batch(&blobs, &commitments, &proofs, &kzg_settings)
-                .unwrap_err();
+        let blobs_copy: Vec<Blob> = blobs.iter().map(|b| *b.clone()).collect();
+        let error = KZGProof::verify_blob_kzg_proof_batch(
+            &blobs_copy,
+            &commitments,
+            &proofs,
+            &kzg_settings,
+        )
+        .unwrap_err();
         assert!(matches!(error, Error::MismatchLength(_)));
 
         let incorrect_blob = generate_random_blob(&mut rng);
         blobs.push(incorrect_blob);
 
+        let blobs_copy: Vec<Blob> = blobs.iter().map(|b| *b.clone()).collect();
```

Comment:
I wish there were a way to convert from `Vec<Box<Blob>>` to `Vec<Blob>` without the blob being stored on the stack intermediately. Unfortunately, I think this is a pretty difficult to do without unsafe code.